docs/artifact_repositories/nuget.md

```diff
@@ -0,0 +1,100 @@
+# Phylum NuGet Registry
+
+The Phylum NuGet registry is based on NuGet's [Server API].
+
+[Server API]: https://learn.microsoft.com/en-us/nuget/api/overview
+
+## Configuration
+
+All configuration options will require a Phylum API key, since Phylum requires
+authentication. You can find out how to generate one in our
+[API Keys documentation].
+
+In the following examples, all API keys will be represented as
+`<PHYLUM_API_KEY>`, so make sure to replace them with your generated key.
+
+Additionally, if the default policy is not sufficient, a group can be passed to
+evaluate all packages against the group's policy. To do this, just replace
+`<PHYLUM_ORG>` and `<PHYLUM_GROUP>` with the desired org and group name. The
+supplied API key **must** have access to this group.
+
+If Phylum's default [policy] is sufficient, you can omit the username from the
+authentication details.
+
+[API Keys documentation]: ../knowledge_base/api-keys.md#generate-an-api-key
+[policy]: ../knowledge_base/policy.md
+
+### `dotnet`
+
+To use the Phylum NuGet registry, the original nuget.org registry first needs to
+be disabled:
+
+```sh
+dotnet nuget disable source nuget.org
+```
+
+Once the official registry is disabled, the Phylum source can be added. If
+you're not running Windows, you'll also have to add the
+`--store-password-in-clear-text` flag.
+
+```sh
+dotnet nuget add source http://localhost:8080/v3/index.json \
+    --protocol-version 3 \
+    --name Phylum \
+    --valid-authentication-types basic \
+    --username <PHYLUM_ORG>/<PHYLUM_GROUP> \
+    --password <PHYLUM_API_KEY>
```

Comment:
Annoyingly, this command only works for Windows users with a Unix-like shell or Linux users running Mono (uncommon). Otherwise either you'll get an error about password encryption or you'll get an error about `--protocol-version` being an invalid command and an incorrectly added Aviary source. CMD and PowerShell both treat `\` as a regular character, and each have their own different escape characters that can be used for line continuations.

Does our documentation site support those tabbed code examples where you can switch languages for the same code block? We could add a PowerShell one here (replace `\` with `` ` ``). It should be the same syntax for PowerShell (the cross-platform one) or Windows PowerShell (the old one that comes preinstalled on Windows).

Reply:
Even on Windows most developers that work with CLIs are using linux shells these days, so I don't consider this much of an issue.

Anyone copy/pasting stuff from online documentation should be able to make this work for their shell anyway. There might be unix shells that don't support `\`, but users can be expected to understand and account for that.

Reply:
On Windows I always use PowerShell. When I was working on Windows software lot of coworkers would stick to CMD and batch files and I could never understand why.

Reply:
> On Windows I always use PowerShell. When I was working on Windows software a lot of coworkers would stick to CMD and batch files and I could never understand why.

Yeah honestly I feel like most of these types of developers are just going to go with the Visual Studio route, so we should figure out if that works with the current Aviary implementation and document that.